src/lib/evas/meson.build

```
@@ -4,11 +4,9 @@
 # as 'source :', later everything is build as libevas.so.
 #
 
-if not sys_windows
-  png = dependency('libpng')
-else
-  png = declare_dependency()
-endif
+png = dependency('libpng',
+  fallback: ['libpng', 'png_dep']
+)
```

I think there is no need to use 3 lines here:
```suggestion
png = dependency('libpng', fallback: ['libpng', 'png_dep'])
```